nhsn/delphi_nhsn/pull.py

```diff
@@ -144,24 +210,31 @@ def pull_preliminary_nhsn_data(
     pd.DataFrame
         Dataframe as described above.
     """
+    # Pull data from Socrata API
```

Comment:
issue: `pull_preliminary_nhsn_data` and `pull_nhsn_data` are really similar. I think it will become a maintenance issue to keep both. We should probably keep these two functions as wrappers of a shared fn that takes a `is_prelim` flag (or similar).

Diff of the two fns:

<img width="1322" alt="Screen Shot 2025-02-11 at 15 20 55" src="https://github.com/user-attachments/assets/bc0a2aaa-20a1-451b-91e0-5967e104eb98" />